azure/azure-release-template.yml

```diff
@@ -65,62 +69,74 @@ extends:
           export PACT_PROVIDER_URL=https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH
 
           if [[ "$APIGEE_ENVIRONMENT" == *"sandbox"* ]]; then
-            export PACT_PROVIDER=nhsd-apim-eps-sandbox
+          export PACT_PROVIDER=nhsd-apim-eps-sandbox
           else
-            export PACT_PROVIDER=nhsd-apim-eps
-            if [ "$APIGEE_ENVIRONMENT" == "int" ]
-            then
+          export PACT_PROVIDER=nhsd-apim-eps
+          if [ "$APIGEE_ENVIRONMENT" == "int" ]
+          then
               export IDP_URL="https://nhsd-apim-testing-$APIGEE_ENVIRONMENT-ns.herokuapp.com"
               docker pull artronics/nhsd-login-docker:latest > /dev/null
               export APIGEE_ACCESS_TOKEN=$(docker run --rm artronics/nhsd-login-docker:latest "$IDP_URL")
-            elif [ "$APIGEE_ENVIRONMENT" == "ref" ]
-            then
+          elif [ "$APIGEE_ENVIRONMENT" == "ref" ]
+          then
               export IDP_URL="$(REF_IDP_URL)"
               export AUTH_BEARER_TOKEN="$(REF_AUTH_BEARER_TOKEN)"
               docker pull booshi/nhsd-login-docker:latest > /dev/null
-              export APIGEE_ACCESS_TOKEN=$(docker run --rm -e AUTH_BEARER_TOKEN=$AUTH_BEARER_TOKEN booshi/nhsd-login-docker:latest "$IDP_URL")
-            else
+              export APIGEE_ACCESS_TOKEN=$(docker run --rm -e AUTH_BEARER_TOKEN="$AUTH_BEARER_TOKEN" booshi/nhsd-login-docker:latest "$IDP_URL")
+          else
               export IDP_URL="https://nhsd-apim-testing-$APIGEE_ENVIRONMENT.herokuapp.com"
               docker pull artronics/nhsd-login-docker:latest > /dev/null
               export APIGEE_ACCESS_TOKEN=$(docker run --rm artronics/nhsd-login-docker:latest "$IDP_URL")
-            fi
+          fi
           fi
 
           # Publish
-          cd $SERVICE_NAME/$SERVICE_ARTIFACT_NAME/pact
+          cd "$SERVICE_NAME"/"$SERVICE_ARTIFACT_NAME"/pact
```

Comment:
Not a big deal by any means, but we could just enclose the entire thing in double quotes instead of having two sets